projects/packages/forms/src/dashboard/class-dashboard.php

```diff
@@ -98,7 +98,11 @@ public function load_admin_scripts( $hook ) {
 
 		wp_add_inline_script(
 			self::SCRIPT_HANDLE,
-			'window.jetpackFormsData = ' . wp_json_encode( array( 'apiRoot' => $api_root ) ) . ';',
+			'window.jetpackFormsData = ' . wp_json_encode(
+				array(
+					'apiRoot' => $api_root,
+				)
+			) . ';',
```

Comment:
Compeltely optional, but since you're touching this anyway it might be worthwhile to pass flags `JSON_UNESCAPED_SLASHES | JSON_HEX_TAG | JSON_HEX_AMP` to avoid a bunch of unnecessary `\/` in the output.